src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/BeneficialOwnerGovernmentOrPublicAuthorityDao.java

```diff
@@ -35,6 +36,18 @@ public class BeneficialOwnerGovernmentOrPublicAuthorityDao {
     @Field("non_legal_firm_members_nature_of_control_types")
     private List<NatureOfControlType> nonLegalFirmMembersNatureOfControlTypes;
 
+    @Field("non_legal_firm_control_nature_of_control_types")
+    private List<NatureOfControlType> nonLegalFirmControlNatureOfControlTypes;
+
+    @Field("trust_control_nature_of_control_types")
+    private List<NatureOfControlType> trustNatureOfControlTypes;
```

Comment:
trustNatureOfControlTypes is missing 'Control' in the attribute name (line 43) - trustControlNatureOfControlTypes

Reply:
Does this mean we need to add "control" to the existing:
TRUSTEES_NATURE_OF_CONTROL_TYPES_FIELD
as well? This is what I took my lead from in naming this field.
I only added the extra control to firm to distinguish it clearly from the member field.

Reply:
perhaps, i'm not sure.
But the field coming in via json will be
"trust_control_nature_of_control_types": {
            "type": "array",
            "items": {
              "$ref": "#/components/schemas/NatureOfControlType"
            }

if we leave it as trustNatureOfControlTypes, then that reads as if it is all the Nocs for the trust

Reply:
In light of the json name this has been updated.